各种接口总报这个错误

```
[2016-05-05 17:08:35.164] [ERROR] oss-logger - TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.extname (path.js:1429:5)
    at Client.proto.createRequest (/Users/madoka/Workspace/github/nodeoss/app/node_modules/ali-oss/lib/client.js:239:65)
    at Client.proto.request (/Users/madoka/Workspace/github/nodeoss/app/node_modules/ali-oss/lib/client.js:296:24)
    at next (native)
    at onFulfilled (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:65:19)
    at /Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:54:5
    at co (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:50:10)
    at toPromise (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:118:63)
    at next (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:99:29)
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.extname (path.js:1429:5)
    at Client.proto.createRequest (/Users/madoka/Workspace/github/nodeoss/app/node_modules/ali-oss/lib/client.js:239:65)
    at Client.proto.request (/Users/madoka/Workspace/github/nodeoss/app/node_modules/ali-oss/lib/client.js:296:24)
    at next (native)
    at onFulfilled (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:65:19)
    at /Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:54:5
    at co (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:50:10)
    at toPromise (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:118:63)
    at next (/Users/madoka/Workspace/github/nodeoss/app/node_modules/co/index.js:99:29)
```

给个思路这么调试

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,7 @@
 sudo: false
 language: node_js
 node_js:
+  - '6'
   - '5'
   - '4'
 env:

--- a/lib/client.js
+++ b/lib/client.js
@@ -236,7 +236,7 @@ proto.createRequest = function (params) {
     if (params.mime && params.mime.indexOf('/') > 0) {
       headers['Content-Type'] = params.mime;
     } else {
-      headers['Content-Type'] = mime.lookup(params.mime || path.extname(params.object));
+      headers['Content-Type'] = mime.lookup(params.mime || path.extname(params.object || ''));
     }
   }
 

